pymc3/model.py

```diff
@@ -158,9 +158,15 @@ def logpt(self):
         return t.sum(self.logp_elemwiset)
 
 class Model(Context, Factor):
-    """Encapsulates the variables and likelihood factors of a model."""
+    """Encapsulates the variables and likelihood factors of a model.
+    
+    Parameters
+    ----------
+    verbose : int
+        Setting for model feedback verbosity (Defaults to 1 -- normal verbosity).
+    """
 
-    def __init__(self):
+    def __init__(self, verbose=1):
```

Comment:
At the moment it is evaluated as if `True`, but we may eventually want to have 2-3 verbosity levels, perhaps for debugging.